features/git-town/configuration/configuration_reset.feature

```diff
@@ -0,0 +1,36 @@
+Feature: resetting the configuration
+
+  As a user wishing to remove all Git Town settings from a repository
+  I want to be able to cleanly reset the Git Town configuration with a single command
+  So that I can continue working on actual work and stay productive
```

Comment:
Okay, what is the reason for this command to exist then? Why would I ever reset the Git Town configuration if not to provide new configuration right afterwards? One cannot use Git Town without configuration, right?